extensions/objects/templates/filepath-editor.directive.ts

```diff
@@ -769,6 +769,8 @@ angular.module('oppia').directive('filepathEditor', [
             filename: ctrl.generateImageFilename(
               dimensions.height, dimensions.width, imageType)
           }));
+          ctrl.entityType = ContextService.getEntityType();
```

Comment:
This also happens at line 876 and 877. Shouldn't we just have one occurrence in this directive?

Reply:
Done, removed.